lib/services/user_service.dart

```diff
@@ -1,13 +1,16 @@
 import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:project_fomo/models/user_data.dart';
+import 'package:project_fomo/utils/structures/response.dart';
 
 class UserService {
   final CollectionReference _userDataCollection =
       Firestore.instance.collection('users');
 
+  static final _databaseReference = Firestore.instance;
```

Comment:
You have access to the users collection in the line above through _userDataCollection.  No need for a new firestore instance here.